DataAccess/DataAccess.SQLite/LiteAdapter.hpp

```diff
@@ -33,8 +45,30 @@ class LiteAdapter : public IAdapter
 
     virtual ~LiteAdapter() {}
 public:
+    /** @brief Method for executing SQL quries.
+     *   @details You shouldn't use this method because it's
+     *    low level accessing the database. Use it if you
+     *    want something specific from database, instead use LiteTable.
+     *    If you want to insert some strings, big text
+     *    or a timestamp you must wrap the string/text by single quotes.
+     *    You don't have to put ';' at the end of query.
+     *   @code
+     *   ...->query("SELECT * FROM table_name WHERE name = 'Alex'");
+     *   @endcode
+     *   @param SQL query in the form of string.
+     *   @return Optional result.
+     */
     std::optional<std::any> query(const std::string_view& query) override;
+
+    /* @brief Method for checking the connection to the db.
```

Comment:
Plz, add another one * in this place.
And add this and all other lines on new line break.
/**
* @brief
like this, if it affects the documentation